.github/workflows/bicep_deploy.yml

```diff
@@ -0,0 +1,137 @@
+        required: true
+      APIM_TFS_URL:
+        description: webapp environment variable
+        required: true
+      APIM_MDM_KEY:
+        description: webapp environment variable
+        required: true
+      APIM_MDM_URL:
+        description: webapp environment variable
+        required: true
+      APIM_MDM_VALUE:
+        description: webapp environment variable
+        required: true
+      CORS_ORIGIN:
+        description: webapp environment variable
+        required: true
+      APIM_ESTORE_URL:
+        description: webapp environment variable
+        required: true
+      APIM_ESTORE_KEY:
+        description: webapp environment variable
+        required: true
+      APIM_ESTORE_VALUE:
+        description: webapp environment variable
+        required: true
+      COMPANIES_HOUSE_API_KEY:
+        description: webapp environment variable
+        required: true
+      ORDNANCE_SURVEY_API_KEY:
+        description: webapp environment variable
+        required: true
+      GOV_NOTIFY_API_KEY:
+        description: webapp environment variable
+        required: true
+      GOV_NOTIFY_EMAIL_RECIPIENT:
+        description: webapp environment variable
+        required: true
+      AZURE_PORTAL_EXPORT_FOLDER:
+        description: webapp environment variable
+        required: true
+      AZURE_PORTAL_FILESHARE_NAME:
+        description: webapp environment variable
+        required: true
+      JWT_SIGNING_KEY:
+        description: webapp environment variable
+        required: true
+      JWT_VALIDATING_KEY:
+        description: webapp environment variable
+        required: true
+      UKEF_INTERNAL_NOTIFICATION:
+        description: webapp environment variable
+        required: true
+      DTFS_CENTRAL_API_KEY:
+        description: webapp environment variable
+        required: true
+      EXTERNAL_API_KEY:
+        description: webapp environment variable
+        required: true
+      PORTAL_API_KEY:
+        description: webapp environment variable
+        required: true
+      TFM_API_KEY:
+        description: webapp environment variable
+        required: true
+      UKEF_TFM_API_SYSTEM_KEY:
+        description: webapp environment variable
+        required: true
+      UKEF_TFM_API_REPORTS_KEY:
+        description: webapp environment variable
+        required: true
+      TFM_UI_URL:
+        description: webapp environment variable
+        required: true
+      AZURE_NUMBER_GENERATOR_FUNCTION_SCHEDULE:
+        description: webapp environment variable
+        required: true
+      SESSION_SECRET:
+        description: webapp environment variable
+        required: true
+      ESTORE_URL:
+        description: webapp environment variable
+        required: true
+      
+jobs:
+  validate:
+    uses: ./.github/workflows/bicep_az_deployment_group.yml
+    with:
+      environment: ${{ inputs.environment }}
+      resourceGroupName: ${{ inputs.resourceGroupName }}
+      action: validate
```

Comment:
Validate is pretty handy - it will check permissions etc, and should give a good idea whether it will deploy correctly.
There is potential to use the `what-if` deploy instead (and I've implemented it), put the output on the PR and block the workflow until it has been approved.